doc/source/train/user-guides/using-gpus.rst

```diff
@@ -127,6 +163,28 @@ You can get the associated devices with :meth:`ray.train.torch.get_device`.
     trainer.fit()
 
 
+Setting the communication backend
+---------------------------------
+You can set the PyTorch distributed communication backend (e.g. GLOO or NCCL) by passing a
+:class:`~ray.train.torch.TorchConfig` to the :class:`~ray.train.torch.TorchTrainer`.
```

Comment:
Can we have some more guidance here, e.g. who might this apply to? One way to to think about  it is - if someone is looking for how to use Gloo they should be able to find this, but if someone doesn't know/care about using Gloo they would not get stuck on this section.